### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by rishiosaur**
Here's an idea for a new weapon and encounter.

### CODE REVIEW

File: script/world.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ var World = {
 			cooldown: 2
 		},
 		'bone spear': {
-			verb: _('stab'),
+			verb: _('throw'),
```
Comment (groteworld) [inline script/world.js]: How about 'lunge'

File: script/room.js [hunk 2]
```diff
@@ -331,9 +331,43 @@ var Room = {
 					'sulphur': 50
 				};
 			}
+		},
+		'i dagger': {
+			name: _('dagger'),
+			type: 'weapon',
+			buildMsg: _("good for melee"),
+			cost: function() {
```
Comment (groteworld) [inline script/room.js]: These daggers seem a little cheap for being 1 damage below the swords and the fact that they add another weapon to stack damage on

File: script/path.js [hunk 3]
```diff
@@ -11,32 +11,35 @@ var Path = {
 		'bullets': 0.1,
 		'energy cell': 0.2,
 		'laser rifle': 5,
-		'bolas': 0.5
+		'bolas': 0.5,
+		'dagger': 1,
+		's dagger': 1,
+		'bow': 5
```
Comment (groteworld) [inline script/path.js]: arrows weigh nothing?

File: script/events/encounters.js [hunk 4]
```diff
@@ -396,5 +396,41 @@ Events.Encounters = [
 				notification: _('a shot rings out, from somewhere in the long grass')
 			}
 		}
+	},
+	{
+		title: _("A madman"),
```
Comment (groteworld) [inline script/events/encounters.js]: Reasoning to add a new encounter?

maybe turn this to an earlier encounter that's easier but provides a `i knife`

**Comment by Continuities** [comment]
I'm inclined to agree with @groteworld. Providing three new weapons in the early-game will drastically simplify a lot of the earlier encounters. If we want to add them, we'll need to tweak the hp of the enemies. What do the daggers provide that the swords don't?

File: script/world.js (hunk 1 above)
Comment (rishiosaur) [reply script/world.js]: Yeah. that'll work. I didn't know that many verbs for it, so I just used that one.

File: script/room.js (hunk 2 above)
Comment (rishiosaur) [reply script/room.js]: Yeah, I guess so. You can increase the costs of them.

**Pull request closed without merging**